### PR TITLE
fix: closes all connections when stopped

### PR DESCRIPTION
**Pull request by jaulz**
This fix closes all connections when the Hapi server is stopped. Otherwise the node process will continue running until the last connection was closed.

### CODE REVIEW

**Review by Nargonath — CHANGES_REQUESTED** [review]
The travis ci build failed. Please check travis log there are some code format issues (missing semi-colons and so on) plus you'll need to add some tests to cover the new code you added. The coverage was at 100% on last build and usually there is a 100% coverage on all module in HapiJS org.

**Comment by jaulz** [comment]
Yep, will tackle this tomorrow. Just wanted to get a feeling first if any PR is welcome since the latest welcome is quite old 😊

**Comment by Nargonath** [comment]
Thanks for your contribution. PR are always welcomed 😉. However I'm not the lead maintainer on this project. I intervened because there were some more work required to make the PR mergeable but in the end @geek will have the final word when it comes to accept or not the contribution.

**Comment by jaulz** [comment]
@Nargonath @geek I created tests for the new feature but in Node.js v11 some strange things happen with the experimental microtask queue. any idea how we could fix this or if we can ignore this in the meantime?

**Comment by Nargonath** [comment]
I think there is a problem with Node 11 at the moment. Perhaps @geek just need to disable the Node 11 job for now as it's been done in https://github.com/hapijs/nes/issues/257.

**Review by Nargonath — APPROVED** [review]
I approve the PR based on my previous review because code is now 100% coverage and lint errors have been fixed however someone with more knowledge of the project should have a look at the feature itself.

**Comment by jaulz** [comment]
Great, thanks! Looking forward for @geek feedback 😊

**Review by geek — APPROVED** [review]

**Comment by geek** [comment]
@jaulz I'll update master to use different node versions on travis. Thanks for the fix.

**Pull request merged**
